src/main/java/com/vamae/statistic/model/Statistic.java

```diff
@@ -0,0 +1,25 @@
+package com.vamae.statistic.model;
+
+import com.vamae.authorization.model.User;
+import jakarta.persistence.Id;
+import lombok.*;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Statistic {
+    @Id
+    private String id;
+    private User user;
+
+    public Statistic(String username, int authCount) {
```

Comment:
You should use bulider agains this constructor, so constructor is redundant. Anyway, constructor must be written after all fields